app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -146,7 +153,7 @@ def cloud_volume_snapshots
   end
 
   def images
-    collector.images.each do |image|
+    collector.images.reject { |img| skip_image?(img) }.each do |image|
```

Comment:
NOTE normally this is done in the collector class, but that would involve duplicating the logic for a deprecated image as well as parsing what image is used by an instance